src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -1,66 +1,103 @@
     }
 
-    @And( "^I navigate to the Mobile Chapter page$" )
-    public void iNavigateToTheFullStackPage(){
+    @And("^I navigate to the Mobile Chapter page$")
+    public void iNavigateToTheMobileChapterPage() {
         ValoriHomePage valoriHomePage = new ValoriHomePage(driver);
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         valoriHomePage.clickBusinessDropdown();
         valoriHomePage.clickMobileInDropDOwn();
-        assertTrue("Expecting the full stack header to be visible", mobileChapterPage.isMobilePageVisibile());
+        assertTrue("Expecting the Mobile automation header to be visible", mobileChapterPage.isMobilePageVisibile());
     }
 
-    @Then( "^I can read articles about the Mobile Chapter$" )
-    public void iCanReadAboutMobileChapter(){
+    @Then("^I can read articles about the Mobile Chapter$")
+    public void iCanReadAboutMobileChapter() {
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         assertTrue(mobileChapterPage.areArticlesVisible());
     }
 
-    @And( "^I contact Valori$" )
-    public void iContactValori()throws Throwable{
+    @And("^I contact Valori$")
+    public void iContactValori() throws Throwable {
         MobileChapterPage mobileChapterPage = new MobileChapterPage(driver);
         ContactUsPage contactUsPage = new ContactUsPage(driver);
         mobileChapterPage.clickContactUsButton();
+        driver.navigate().refresh();
 
-        //switch to new tab
+        //switch to new tab.
+        // Dit zorgt ervoor dat er een tab wordt geopend naar innovation.valori.nl/neem-contact-op
+        //Echter moet de link https://innovation.valori.nl/neem-contact-op zijn om de pagina te bereiken.
         ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
         driver.switchTo().window(tabs.get(1));
+
+        //nakijken of de juiste pagina is bereikt.
         assertTrue(contactUsPage.isTitleVisible());
     }
 
-    @Then( "^the contact us form is available$" )
+    @Then("^the contact us form is available$")
     public void theContactUsFormIsAvailable() throws Throwable {
-        //TODO IMPLEMENT THIS
-        //Verify the contact form fields are available.
-        throw new PendingException();
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        //nakijken of het contactformulier aanwezig is.
+        assertTrue(contactUsPage.isFormAvailable());
+
     }
-}
 
+    @Given("^fill the contact form with the following data$")
+        public void fillContactForm(DataTable exampleTable) throws Throwable {
+            UtilitiesAka aka = new UtilitiesAka(driver);
+        ContactUsPage contactUsPage = new ContactUsPage(driver);
+        //Nakijken of de contactformulier velden aanwezig zijn en deze ook gelijk invullen.
+        List<Map<String, String>> list = exampleTable.asMaps(String.class, String.class);
```

Comment:
Je gaat hier alle data uit je datatable achter elkaar invullen, wat in feite losse testcases zouden moeten zijn.  Je hebt in dit geval een data table gebruikt die je in kunt zetten om een heel formulier in 1 keer in te vullen. Wil je nou een formulier in vullen met verschillende data sets gebruik dan een example table. Dan voert hij het volledige scenario nogmaals uit met andere data en geeft je dan ook betere rapportage.